src/Api/TMDB/tmdbService.tsx

```diff
@@ -0,0 +1,91 @@
+export const fetchMovieGenres = async () => {
+    const options = {
+        method: "GET"
+    }
+    const url = `${import.meta.env.VITE_TMDB_URL}genre/movie/list?api_key=${import.meta.env.VITE_TMDB_APIKEY}`;
```

Comment:
can just create a top level variable for `import.meta.env.VITE_TMDB_URL` instead of writing it again and again.  Same for all the env variables you are using.

Reply:
Created a top level variables for all the environment variables